content/docs/forms.md

```diff
@@ -116,35 +116,35 @@ class EssayForm extends React.Component {
           Essay:
           <textarea value={this.state.value} onChange={this.handleChange} />
         </label>
-        <input type="submit" value="Submit" />
+        <input type="submit" value="Gönder" />
       </form>
     );
   }
 }
 ```
 
-Notice that `this.state.value` is initialized in the constructor, so that the text area starts off with some text in it.
+`this.state.value` 'in constructor’te başlatıldığına dikkat edin, böylece textarea içerisinde varsayılan olarak bu yazı bulunacaktır
```

Comment:
`this.state.value` 'in constructor’te başlatıldığına dikkat edin
yerine
`this.state.value`'nun constructor’da başlatıldığına dikkat edin
daha uygundur.